app/controllers/v0/health_care_applications_controller.rb

```diff
@@ -18,8 +18,6 @@ def create
       result = begin
         HCA::Service.new(current_user).submit_form(form)
       rescue Common::Client::Errors::ClientError => e
-        log_exception_to_sentry(e)
```

Comment:
can you leave this in and change `skip_sentry_exception_types` to be
```ruby
def skip_sentry_exception_types
  super + [Common::Exceptions::GatewayTimeout, Common::Exceptions::BackendServiceException]
end
```

Reply:
I'll make this change, but I'm a little concerned since this controller could, in the future, make use of another `service` class where `Common::Exceptions::BackendServiceException` would be cause for concern.

Reply:
we're moving towards doing all external requests in background jobs so it shouldn't be a problem